docker-entrypoint.sh

```diff
@@ -13,6 +13,34 @@ rm -f /fgc/data/browser/SingletonLock
 # ls -l /tmp/.X11-unix/
 rm -f /tmp/.X1-lock
 
+# Check and export secrets to variables if exist
+# Get list of VARIABLES with "_FILE" at the end
+SECRETS_LIST=$(env | grep "_FILE")
```

Comment:
I thought also to limit those to only credentials and add here whitelist, e.g. `PASSWORD_FILE`, `EMAIL_FILE`, `OTP_FILE`.
```
grep -E "PASSWORD_FILE|EMAIL_FILE|OTP_FILE"
```